docs/blog/what-to-expect-as-a-speaker.rst

```diff
@@ -0,0 +1,53 @@
+
+During the conference
+---------------------
+
+**Sign up for and attend a sound check.**
+  Between each other, attendees, and speakers, the organizers of the conference are basically herding cats. Which is something we’re probably all more or less familiar with. We’re also familiar with the initial 2 minutes of any video call being spent on “hold on, can you hear me? My mic’s not working”. To avoid this happening during your live Q&A session, you’ll be asked to book a slot with one of the organizers and do a camera and sound check with them in the actual software used for the conference. It’s a good little ice breaker, too.
+**Your talk! Plus a Q&A session.**
+  Your talk will be show at a given time (in the local conference timezone). Immediately after, there’ll be a short Q&A session in where you’re live with an emcee who’ll funnel questions to you from the chat (as well as any they might have themselves). This is where the sound check comes in. This is the only time you’re officially expected to be present and engaging with others. 
+  
+  In some of the Write the Docs documentation about conferences, you might find this phrase or something similar; that speakers are expected to actively participate in the conference. This is, in some ways, a leftover from when conferences were in-person events. For virtual conferences, when I asked, the messaging was this; attend, watch as many talks and be in as many unconference sessions as you want, be there and present for your Q&A; but you don’t have to “perform” or be “visible” for other peoples’ talks etc. You CAN, of course! But let your mental capacity and energy be your guide.
+  
+  Now, back to the Q&A; prepare to have fun with this! You effectively “gave” your talk a month ago, because you recorded it and sent it to someone else, only for it to be shown a set time to a large group of other people. And they’re all reacting to it at the same time! So, now is when you get to reflect, talk, answer weird and lovely questions. It can be super enjoyable and stimulating. Also, just FYI, this part will be appended to your actual talk when the recording goes on up YouTube.
+**Unconference sessions.**
+  During the conference, participants can schedule unconference schedules to run in the same timeslots as talks. Sometimes, this’ll cause clash of interests; you might want to hear that one particular talk and be in the Q&A as it’s happening, but also in that interesting discussion that spawned because of that *other* talk, where you just definitely need to dig into that one point. Stimulating!
```

Comment:
```suggestion
  During the conference, participants can schedule unconference schedules to run in the same timeslots as talks. Sometimes, this’ll cause a clash of interests; you might want to hear that one particular talk and be in the Q&A as it’s happening, but also in that interesting discussion that spawned because of that *other* talk, where you just definitely need to dig into that one point. Stimulating!
```